R/import_subject.R

```diff
@@ -1,25 +1,28 @@
 #' function for reading multimodal data files for a subject according to a set of parsing routines. This function is in working shape.
 #' @importFrom checkmate assert_subset assert_file_exists
 #' @export
-import_subject <- function(files, parsers) {
+import_subject <- function(files, yaml_file) {
```

Comment:
Under the approach of passing in the yaml, does the user get the option of passing in a parser function name for the data?